/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:306: RuntimeWarning: invalid value encountered in true_divide
  assigned_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi), axis = 1) / np.sum(mat_y_hpi, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:322: RuntimeWarning: invalid value encountered in true_divide
  closest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_closest), axis = 1) / np.sum(mat_y_hpi_closest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:338: RuntimeWarning: invalid value encountered in true_divide
  farthest_dist_hpi = np.nan_to_num(np.sum(np.multiply(C, mat_y_hpi_farthest), axis = 1) / np.sum(mat_y_hpi_farthest, axis = 1), posinf=0)
/export/storage_covidvaccine/Code/utils/create_row.py:343: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi = sum(CA_TRACT['Dist_HPI_weighted'].values) / sum(CA_TRACT['Farthest_Population_HPI'].values)
/export/storage_covidvaccine/Code/utils/create_row.py:344: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi1 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 1]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:345: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi2 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 2]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:346: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi3 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 3]['Farthest_Population_HPI'])
/export/storage_covidvaccine/Code/utils/create_row.py:347: RuntimeWarning: invalid value encountered in double_scalars
  farthest_dist_hpi4 = sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Dist_HPI_weighted'].values) / sum(CA_TRACT[CA_TRACT['HPIQuartile'] == 4]['Farthest_Population_HPI'])
